pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -223,9 +275,15 @@ type cpuAccumulator struct {
 	result cpuset.CPUSet
 
 	numaOrSocketsFirst numaOrSocketsFirstFuncs
+
+	// sortCPUsPackedOrSpread is used to control the cpu sorting result.
+	// The sequence of returned CPU IDs depends on the policy.
+	// By default, cpus is sorted by sortAvailableCPUsPacked()
+	// If packed is false, cpu is sorted by sortAvailableCPUsSpread()
+	sortCPUsPackedOrSpread sortCPUsPackedOrSpreadFuncs
```

Comment:
Following above, this variable would then become:
```
availableCPUSorter availableCPUSorter
```

Reply:
updated